### PR TITLE
Update the Welcome page

### PR DESCRIPTION
**Pull request by mhairimca**

### CODE REVIEW

File: welcome.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
```
Comment (Tommy1231232) [inline welcome.md:1]: @mhairimca it looks like you added a duplicate welcome page (see below for all of the content you added). Can you update (all items highlighted in green is what is added)

File: welcome.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Because the DVM requires 2-4 days to respond to a price request, it is not intended to be used as an on-chain price feed that pushes prices to financial contracts that need it. Rather, it is complementary to “priceless” financial contracts.
+
+The DVM is designed to include an economic guarantee around its cost of corruption and profit from corruption. The cost of corrupting the DVM, as measured by the cost of 51% of the UMA voting tokens, should be greater than the profit from corrupting the DVM, as measured by the collateral stored in the financial contracts that are registered with it. To ensure that this inequality holds, the DVM may charge fees to financial contracts that are used to raise the price of the UMA voting tokens.
+
+Governance is provided by the UMA Improvement Proposals (UMIPs) which allow new collateral types and price identifiers to be added to the system as well as new types of factory contract.
+
+
+**How UMA works**
```
Comment (Tommy1231232) [inline welcome.md:75]: We will not want this on the welcome page. This is for understanding UMA and when users go to the overview page they will be able to be directed to this tab

File: welcome.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+<!--taken from UMA/TRS Mission and Values -->
+Our mission is to make financial markets universally fair and accessible.
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
+##What is UMA##
```
Comment (Tommy1231232) [inline welcome.md:52]: This is in the overview page. No need to be on a welcome page. The What infrastructure does uma provide section should suffice. This is intended to be a navigation site to direct people to the right place

File: welcome.md (hunk 2 above)
Comment (mhairimca) [reply welcome.md]: I've removed it, however I'm surprised that a protocol would not want a reference to governance on its welcome page.

File: welcome.md (hunk 1 above)
Comment (mhairimca) [reply welcome.md:1]: now deleted

File: welcome.md (hunk 2 above)
Comment (Tommy1231232) [reply welcome.md]: This comment was related to detailing "How UMA Works". Feel free to leave in a blurb on governance

File: welcome.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Because the DVM requires 2-4 days to respond to a price request, it is not intended to be used as an on-chain price feed that pushes prices to financial contracts that need it. Rather, it is complementary to “priceless” financial contracts.
+
+The DVM is designed to include an economic guarantee around its cost of corruption and profit from corruption. The cost of corrupting the DVM, as measured by the cost of 51% of the UMA voting tokens, should be greater than the profit from corrupting the DVM, as measured by the collateral stored in the financial contracts that are registered with it. To ensure that this inequality holds, the DVM may charge fees to financial contracts that are used to raise the price of the UMA voting tokens.
+
+
+## How UMA works ##
```
Comment (Tommy1231232) [inline welcome.md:37]: I'm ok with having a more detailed "What is UMA" section on the welcome page but having How UMA works is a bit too much as it ruins the flow of having this page be a spot for various stakeholders to be able to quickly be directed to content that relates to them (token holders, developers, new comers)

File: welcome.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## What is UMA ##
+
+UMA builds open source infrastructure for developers to create financial instruments in a permissionless, secure and efficient manner. 
+
+This infrastructure includes:
+ - [Priceless financial contract templates](getting-started/priceless-financial-contracts.md) which can be used to create synthetic tokens
+ - An optimistic oracle known as the [Data Verification Mechanism](getting-started/oracle.md) (DVM) to enforce financial contracts built on UMA
+ - A [Decentralized Governance Process](getting-started/governance.md) to manages improvements of UMA protocol
+
+These components enable the creation of fast, efficient, and secure synthetic assets on Ethereum.
```
Comment (Tommy1231232) [inline welcome.md]: It's the first time we call out synthetic assets. We should link to the section that discusses this in more detail.

File: welcome.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Our mission is to make financial markets universally fair and accessible.
+
+UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a pre-requisite to a free, open and fair financial system. 
+
+UMA's goal is to enable anyone to attain or transfer any form of risk seamlessly and securely through the UMA protocol thus empowering everyone to participate in a universally accessible financial system.
+
+## What is UMA ##
+
+UMA builds open source infrastructure for developers to create financial instruments in a permissionless, secure and efficient manner.
```
Comment (Tommy1231232) [inline welcome.md]: Should we call out synthetic assets here versus financial instruments? Or say how the first area UMA is focusing on is synthetic assets?

File: welcome.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+
```
Comment (Tommy1231232) [inline welcome.md:2]: Hey Mhairi I tried for quite some time to make this work with a more detailed What is UMA section and at the end of it decided it is best to remove it from the welcome page and have it in it's own section.

File: welcome.md (hunk 5 above)
Comment (Tommy1231232) [reply welcome.md]: I see we removed the header for "understanding UMA" for someone who is looking to learn more about uma where should they head?

File: welcome.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+ - [Priceless financial contract templates](getting-started/priceless-financial-contracts.md) which can be used to create synthetic tokens
+ - An optimistic oracle known as the [Data Verification Mechanism](getting-started/oracle.md) (DVM) to enforce financial contracts built on UMA
+
+These components enable the creation of fast, efficient, and secure synthetic assets on Ethereum. 
+
+**For Developers**
+
+If you are a developer interested in building with UMA Protocol, view [this](https://docs.umaproject.org/synthetic-tokens/what-are-synthetic-assets) explainer of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens.
+
+Additionally, the [development guide](https://docs.umaproject.org/build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA.
```
Comment (smb2796) [inline welcome.md:30]:
```suggestion
Follow the [development guide](https://docs.umaproject.org/build-walkthrough/build-process) to launch your own expiring synthetic tokens on UMA.
```

File: welcome.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+This infrastructure includes:
+ - [Priceless financial contract templates](getting-started/priceless-financial-contracts.md) which can be used to create synthetic tokens
+ - An optimistic oracle known as the [Data Verification Mechanism](getting-started/oracle.md) (DVM) to enforce financial contracts built on UMA
+
+These components enable the creation of fast, efficient, and secure synthetic assets on Ethereum. 
+
+**For Developers**
+
+If you are a developer interested in building with UMA Protocol, view [this](https://docs.umaproject.org/synthetic-tokens/what-are-synthetic-assets) explainer of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens.
```
Comment (smb2796) [inline welcome.md:28]:
```suggestion
If you are a developer interested in building with UMA Protocol, view [this](https://docs.umaproject.org/synthetic-tokens/what-are-synthetic-assets) explainer of how UMA's priceless synthetic tokens work.

```

File: welcome.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+Additionally, the [development guide](https://docs.umaproject.org/build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA.
+
+You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](https://docs.umaproject.org/dev-ref/addresses).
+
+**For UMA Tokenholders**
+
+If you are interested in the UMA project token, please read [this section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token and the rights and responsibilities of UMA project tokenholders.
```
Comment (smb2796) [inline welcome.md:36]:
```suggestion
If you are interested in the $UMA governance token, please read [this section](uma-tokenholders/uma-holders.md) to learn more about the rights and responsibilities of $UMA tokenholders.

```

File: welcome.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+**Contacts**
+
+If you have any questions or want to connect with the UMA community, please check out our [community page](community/community-overview), join our [discord](https//discord.umaproject.org) and [discourse](https://discourse.umaproject.org)
```
Comment (smb2796) [inline welcome.md:40]:
```suggestion
If you have any questions or want to connect with the UMA community, please check out our [community page](community/community-overview), join our [Discord](https//discord.umaproject.org) or get involved in conversations on the UMA [Discourse](https://discourse.umaproject.org).

```

File: welcome.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+
+---
+
+UMA stands for "Universal Market Access" 
+
+UMA's mission is to make financial markets universally fair and accessible.
+
+UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open markets enables everyone equal access and footing to pursue prosperity and build their financial independence.  Universal access to financial markets is a pre-requisite to a free, open and fair financial system.
```
Comment (smb2796) [inline welcome.md:12]: This is great!

```suggestion
UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open markets enables everyone equal access and footing to pursue prosperity and build their financial independence. Universal access to financial markets is a pre-requisite to a free, open and fair financial system. 

```

File: welcome.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+---
+
+UMA stands for "Universal Market Access" 
+
+UMA's mission is to make financial markets universally fair and accessible.
+
+UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open markets enables everyone equal access and footing to pursue prosperity and build their financial independence.  Universal access to financial markets is a pre-requisite to a free, open and fair financial system. 
+
+UMA's goal is to enable anyone to attain or transfer any form of risk seamlessly and securely through UMA protocol thus empowering everyone to participate in a universally accessible financial system.
```
Comment (smb2796) [inline welcome.md:14]:
```suggestion
UMA's purpose is to enable anyone to attain or transfer any form of risk seamlessly and securely, thus empowering anyone to participate in a universally fair and accessible financial system.

```

**Pull request closed without merging**